k8s/main.ts

```diff
@@ -95,6 +95,13 @@ export class MyChart extends PennLabsChart {
       secret: clubsSecret,
       cmd: ["python", "manage.py", "expire_membership_invites"],
     });
+
+    new CronJob(this, 'graduate-users', {
+      schedule: cronTime.everyYearIn(1, 1, 12, 0),
```

Comment:
I don't think this is right. `graudate_users` graduates all users with year <= the current year, so running this at the beginning of the year will graduate people prematurely (in January instead of May).